files/faban/src/cloud/benchflow/collectors/faban.go

```diff
@@ -67,9 +67,9 @@ func backupHandler(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		panic(err)
 		} 
-	minioKey := minio.GenerateKey(path)
+	minioKey := minio.GenerateKey("faban")
```

Comment:
@VincenzoFerme But that's what I'm doing, I'm generating the key prefix once, and then sending the files with the keys being the prefix plus the actual filename added, which means the files have keys with the same hash for that trial